library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -756,4 +834,27 @@ private GafferPopGraphVariables createVariables() {
         variablesMap.put(GafferPopGraphVariables.SCHEMA, graph.getSchema());
         return new GafferPopGraphVariables(variablesMap);
     }
+
+    /**
+     * Gets the next ID to assign to a supplied vertex based on the currently configured
+     * ID manager.
```

Comment:
This stuff doesn't seem to be very useful. It's required for the TinkerPop suite?